nixos/modules/services/cluster/kubernetes/addons/dashboard.nix

```diff
@@ -46,20 +46,16 @@ in {
     };
 
     image = mkOption {
-      description = "Docker image to seed for the kubernetes dashboard container.";
+      description = "Container image to be used the kubernetes dashboard.";
       type = types.attrs;
       default = {
         imageName = "k8s.gcr.io/kubernetes-dashboard-amd64";
-        imageDigest = "sha256:0ae6b69432e78069c5ce2bcde0fe409c5c4d6f0f4d9cd50a17974fea38898747";
```

Comment:
Naive question: Any reason for not having the imageDigest anymore ?

Reply:
Yep, we can keep it. Reverted that change :+1: